src/agent/onefuzz-agent/src/managed/cmd.rs

```diff
@@ -16,7 +16,7 @@ pub async fn run(args: &clap::ArgMatches<'_>) -> Result<()> {
     let result = config.run().await;
 
     if let Err(err) = &result {
-        error!("error running task: {}", err);
+        error!("error running task: {:?}", err);
```

Comment:
We may as well use this PR to update any other such occurrences of this pattern: `rg -t rust '(error|warn)!.*\{\}'`

Reply:
Per discussion, holding off on updating other logging for moving to `tracing`.